packages/dropdowns/src/elements/Menu/Menu.spec.tsx

```diff
@@ -81,4 +81,23 @@ describe('Menu', () => {
 
     expect(getByTestId('menu').style.width).toBe('100px');
   });
+
+  it('renders menu within appendToNode prop', () => {
+    const { getByTestId } = render(
+      <Dropdown>
+        <Field>
+          <Select data-test-id="select">Example</Select>
+        </Field>
+        <Menu data-test-id="menu" placement="top" appendToNode={document.body}>
+          <Item value="item-1" data-test-id="item">
+            Item 1
+          </Item>
+        </Menu>
+      </Dropdown>
+    );
+
+    userEvent.click(getByTestId('select'));
+
+    expect(getByTestId('menu').parentElement!.parentElement!.tagName).toBe('BODY');
+  });
```

Comment:
Would you be ble to refactor this test to align with the testing-library's [guiding principles](https://testing-library.com/docs/guiding-principles/) and use [priority querying](https://testing-library.com/docs/queries/about/#priority)?.